package.json

```diff
@@ -22,15 +22,19 @@
     "turbolinks": "^5.2.0"
   },
   "prettier": {
```

Comment:
flagging this, this is where all the prettier config currently lives. currently not deviating from the default too much, but open to suggestions.

Reply:
With semicolons, prettier will remove them, but then babel will come along after prettier and add them in the transpiled JS output, is this an accurate understanding?

Reply:
i'm not 100% sure what happens on transpilation. iirc semicolon is optional in the ES spec.

Reply:
that being said, I imagine semicolons come back when minification happens.